### ISSUE TITLE
How to set request mode to 'no-cors'?

### ISSUE THREAD

**Issue by geoidesic** [jshemas/openGraphScraperLite#7]
I'm getting this error:
```
Access to fetch at 'https://www.udemy.com/course/vector-drawing-on-the-ipad-with-affinity-designer/' from origin 'http://localhost:5604' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by jshemas** [comment]
This library no longer does HTTP requests, it will now just pull Open Graph and Twitter Card info from HTML pass into it. Please open a new issue if you are still having problems.
